mithril-aggregator/src/event_store/transmitter_service.rs

```diff
@@ -36,31 +35,10 @@ where
 }
 
 impl TransmitterService<EventMessage> {
-    /// Craft and send an [EventMessage] given the serializable data.
-    /// This method is done in a way to make as simple as possible to send a
-    /// message and make any error not to cause a business failure. A warning is
+    /// Send an [EventMessage].
+    /// This method make any error not to cause a business failure. A warning is
```

Comment:
Maybe we could rephrase slightly to improve readability, something like:
```suggestion
    /// This method prevents any error from causing a business failure. A warning is
```